guides/common/modules/proc_configuring-server-to-synchronize-content-over-a-network.adoc

```diff
@@ -24,31 +33,36 @@ Default is `Library`.
 Default is `Default_Organization_View`.
 . From the *SSL CA Content Credential* menu, select a CA certificate used by the upstream {ProjectServer}.
 . Click *Update*.
-
-The repositories should appear under *Content* > *Products*.
-
-You can synchronize the repositories using the *Sync Now* feature.
-You can also create a synchronization plan to ensure updates on a regular basis. For more information, see {ContentManagementDocURL}Creating_a_Sync_Plan_content-management[Creating a Synchronization Plan] in the _Content Management Guide_.
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click *Sync Now* to synchronize the repositories.
++
+You can also create a synchronization plan to ensure updates on a regular basis.
+For more information, see {ContentManagementDocURL}Creating_a_Sync_Plan_content-management[Creating a Synchronization Plan] in the _Content Management Guide_.
 
 .CLI Procedure
-. Log in to your {ProjectServer} using SSH.
-. Display information about your organization: // TODO upstream or downstream?
+. Connect to your downstream {ProjectServer} using SSH.
+. View information about the upstream CA certificate:
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer organization show --name="_My_Organization_"
+# hammer content-credential show --organization="_My_Downstream_Organization_" \
+--name="_upstream_ca_cert_"
+*Id:           _3_*
+Name:         _upstream_ca_cert_
+Organization: _My_Downstream_Organization_
+Content:      ...
```

Comment:
```suggestion
# hammer content-credential show \
--name="_My_Upstream_CA_Cert_" \
--organization="_My_Downstream_Organization_"
```

We generally avoid interactive shell sessions, i.e. showing commands and their output.

Reply:
Done.